sklearn/dummy.py

```diff
@@ -454,17 +454,26 @@ def predict(self, X):
         X : {array-like, object with finite length or shape}
             Training data, requires length = n_samples
 
+        return_std : boolean, optional
+            Whether to return the standard deviation of posterior prediction.
+            All zeros in this case.
+
         Returns
         -------
         y : array, shape = [n_samples]  or [n_samples, n_outputs]
             Predicted target values for X.
+
+        y_std : array, shape = [n_samples]  or [n_samples, n_outputs]
+            Standard deviation of predictive distribution of query points.
         """
         check_is_fitted(self, "constant_")
         n_samples = _num_samples(X)
 
-        y = np.ones((n_samples, 1)) * self.constant_
+        y = np.ones((n_samples, self.n_outputs_)) * self.constant_
```

Comment:
Why do you change 1 to self.n_outputs_? Seems that the shape of self.constant_ is supposed to be [n_outputs]

Reply:
Oh you're right. The change doesn't make it technically wrong, I thought this made the code easier to read. Should I undo it?

Reply:
Ah I see in your comments, you said I can ignore this comment. Thanks!